### PR TITLE
(PC-12457) fix(UnderageAccountCreated): Fix creditedAmount by adding formatToFrenchDecimal

### PR DESCRIPTION
**Pull request by WendyPassculture**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-12457

## Checklist

I have:

- [x] Made sure the title of my PR follows the [convention](1) `($jira) $type($scope): $summary`.
- [x] Made sure my feature is working on the relevant real / virtual devices (native and web).
- [ ] Written **unit tests** native (and web when implementation is different) for my feature.
- [x] Added a **screenshot** for UI tickets.
- [ ] Attached a **ticket number** or **github dev name** for any added TODO / FIXME.
- [ ] Added new reusable components to **AppComponents** page and communicate to the team on slack.
- [ ] If my PR is a bugfix, I add the link of the "résolution de problème sur le bug" [on Notion](2)
- [ ] Made sure translations file is up to date (`yarn translations:extract`)

## Screenshots

| iPhone |
| --: |
|Capture d’écran 2021-12-22 à 15 02 57|

[1]: https://github.com/pass-culture/pass-culture-app-native/blob/master/doc/standards/pr-title.md
[2]: https://www.notion.so/passcultureapp/R-solution-de-probl-mes-sur-les-bugs-5dd6df8f6a754e6887066cf613467d0a

### CODE REVIEW

**Review by yorickeando — APPROVED** [review]
Ok pour moi !

File: src/locales/fr/messages.po [hunk 1]
```diff
@@ -2895,7 +2896,7 @@ msgstr "Vérifier mon identité"
 
 #: src/libs/i18n.native.test.ts
 msgid "Welcome to Pass Culture"
-msgstr "Bienvenue à Pass Culture"
+msgstr "Welcome to Pass Culture"
```
Comment (antoinewg) [inline src/locales/fr/messages.po:2899]:
```suggestion
msgstr "Bienvenue à Pass Culture"
```

Comment (antoinewg) [reply src/locales/fr/messages.po:2899]: c'est un peu particulier celui là :)

Comment (WendyPassculture) [reply src/locales/fr/messages.po]: C'est modifié.

**Review by antoinewg — APPROVED** [review]

**Pull request merged**